### DISCUSSION TITLE
Builds are queued indefinitely

### DISCUSSION THREAD

**Discussion by huzaifahj** [vercel/community#4250] [Help] [answered]
### Summary

Status is stuck on queued after one failed build

### Steps to Reproduce

N/A

**Comment by theostrahlen** [comment]
I have the same issue right now without any failed builds.

**Comment by huzaifahj** [comment]
Seems like they've made this issue official

<img width="855" alt="image" src="https://github.com/vercel/community/assets/52999758/36b34237-3ce1-4236-8537-7fc53df75fe0">

**Comment by ofhouse** [comment, marked as answer]
We had an issue with builds for Hobby & Pro customers and are actively working on mitigation.
Please see more information on our status page: https://www.vercel-status.com/incidents/87m1ypq44hf0
